examples/cryptorealtime/src/main/java/run/RunThePipeline.java

```diff
@@ -0,0 +1,218 @@
+        bitFinexPair.add(CurrencyPair.ETH_USD);
+        bitFinexPair.add(CurrencyPair.XRP_USD);
+        bitFinexPair.add(CurrencyPair.BCH_USD);
+        /* how to add additional custom pairs?
+        bitFinexPair.add(new CurrencyPair("XTZ", "BTC"));*/
+
+        allExchanges.add(new ExchangeConfiguration(BitfinexStreamingExchange.class.getName(),"bitfinex", bitFinexPair));
+
+
+        // BITSTAMP
+        ArrayList<CurrencyPair> bitStampPair = new  ArrayList<CurrencyPair>();
+        bitStampPair.add(CurrencyPair.BTC_USD);
+        bitStampPair.add(CurrencyPair.ETH_USD);
+
+        allExchanges.add(new ExchangeConfiguration(BitstampStreamingExchange.class.getName(),"bitStamp", bitStampPair));
+
+
+        // OkCOIN
+        ArrayList<CurrencyPair> okCoinPair = new  ArrayList<CurrencyPair>();
+        okCoinPair.add(CurrencyPair.BTC_USD);
+        okCoinPair.add(CurrencyPair.ETH_BTC);
+        okCoinPair.add(CurrencyPair.ETH_USD);
+        okCoinPair.add(CurrencyPair.BCH_BTC);
+        allExchanges.add(new ExchangeConfiguration(OkCoinStreamingExchange.class.getName(),"okCoin", okCoinPair));
+
+
+        // GEMINI
+        ArrayList<CurrencyPair> geminiPair = new  ArrayList<CurrencyPair>();
+        geminiPair.add(CurrencyPair.BTC_USD);
+        geminiPair.add(CurrencyPair.ETH_USD);
+        geminiPair.add(CurrencyPair.BTC_EUR);
+
+        allExchanges.add(new ExchangeConfiguration(GeminiStreamingExchange.class.getName(),"gemini", geminiPair));
+
+        // POLONIEX
+        ArrayList<CurrencyPair> poloniexPair = new  ArrayList<CurrencyPair>();
+        poloniexPair.add(CurrencyPair.BTC_USDT);
+        poloniexPair.add(CurrencyPair.XRP_USDT);
+        poloniexPair.add(CurrencyPair.ETH_BTC);
+
+        allExchanges.add(new ExchangeConfiguration(PoloniexStreamingExchange.class.getName(),"poloniex", poloniexPair));
+
+        // HitBtc
+        ArrayList<CurrencyPair> HitBTCPair = new  ArrayList<CurrencyPair>();
+        HitBTCPair.add(CurrencyPair.BTC_USD);
+        HitBTCPair.add(CurrencyPair.ETH_USD);
+
+        allExchanges.add(new ExchangeConfiguration(HitbtcStreamingExchange.class.getName(),"hitBTC", HitBTCPair));
+
+        // Dispatcher
+        // 1) iterate all exchanges
+        for (ExchangeConfiguration exConf : allExchanges) {
+            // 2) iterate all trading pairs and spawn jobs
+            for(CurrencyPair cuPair : exConf.getListOfPairs()) {
+                PCollection<TradeLoad> ptemp = p.apply(exConf.getExchangeKeyName(),Read.from(new CryptoMarketTradeUnboundedSource(exConf.getExchange(), exConf.getExchangeKeyName(),cuPair)));
+                ptemp
+                        .apply(cuPair.base.toString()+"-"+cuPair.counter.toString()+" Mut",ParDo.of(MUTATION_TRANSFORM))
+                        .apply(cuPair.base.toString()+"-"+cuPair.counter.toString(),CloudBigtableIO.writeToTable(config));
+            }
+        }
+
+
+        // run the pipeline
+        p.run();
+    }
+
```

Comment:
Why are there so many blank lines here? Please remove the extra ones.

Reply:
Done in https://github.com/GoogleCloudPlatform/professional-services/pull/191/commits/05ab1977716987c886d5bb7bb221ea4c72cdc1ee